src/form-builder/CheckboxFieldGroup.tsx

```diff
@@ -12,11 +12,31 @@ const CheckboxFieldGroup = (props: CheckboxGroupProps): JSX.Element => {
   // component re-renders these values per form input inside the form
   const withValidation = {
     ...props,
-    validate: chainValidations(props, [requiredValue]),
+    validate: chainValidations(props, [required]),
   };
   const [field, meta, helpers] = useField(
-    withValidation as FieldHookConfig<string[]>
+    withValidation as FieldHookConfig<boolean>
   );
+
+  const onGroupChange = (event: any) => {
```

Comment:
Just want to make sure, should TSDoc comments be written for this function? https://tsdoc.org/